### ISSUE TITLE
When is the next release? Set up CI job to create releases once a day?

### ISSUE THREAD

**Issue by peterge1998** [LawnchairLauncher/lawnicons#756]
### About this discussion

Hey all, i love this project. I did not receive a helpful answer in the tg group, so i ask here.
When will the next release happen?
I looked into contributing some icons (and app names to app/src/main/res/xml/grayscale_icon_map.xml), but the last release was 1,5 month ago and many were already added...

**Comment by Chefski** [comment]
New release will be coming soon after the contacts, messages & phone icons are redesigned. We'll try to make a new release every 30 days.

**Comment by peterge1998** [comment]
Ok great. Not the solution i prefer but at least i know when i can look into it, thx!

**Comment by SuperDragonXD** [comment]
I'd much prefer having this issue open so I could keep track of it (and maybe implement it soonish, releasing every month). Also the "daily" builds can be seen in the Actions tab (see #Downloads of the README file)

**Comment by luximus-hunter** [comment]
> Also the "daily" builds can be seen in the Actions tab (see #Downloads of the README file)

That's true, but it requires people to unzip the artifacts making the process more complicated that it needs to be. [ALVR](https://github.com/alvr-org/ALVR-nightly) made a separate nightly repository. We could do the same for Lawnicons, and even Lawnchair

**Comment by aDifferentBook** [comment]
> New release will be coming soon after the contacts, messages & phone icons are redesigned. We'll try to make a new release every 30 days.
30 days release cycle is really good option

**Comment by ThatOneCalculator** [comment]
Any update?

**Comment by x9136** [comment]
I suggest automating releases according to the following logic: if a month has passed since the last release and more than 100 icons have been added, then make a release. If less than 100 icons have been added in a month since the last release, then the release is postponed for a month.

**Comment by Hamster45105** [comment]
Until something like this is implemented by the Lawnicons team, I've made my own Lawnicons nightly repository. New releases are made every day (using GitHub actions), as long as there have been changes since the last release happened.

You could add the repo to Obtainium or something similar to automate the downloading of the APKs.

You can access it [here](https://github.com/Hamster45105/lawnicons-nightly).

Hope you find it helpful :)

**Comment by aDifferentBook** [comment]
> Until something like this is implemented by the Lawnicons team, I've made my own Lawnicons nightly repository. New releases are made every day (using GitHub actions), as long as there have been changes since the last release happened.
> 
> You could add the repo to Obtainium or something similar to automate the downloading of the APKs.
> 
> You can access it [here](https://github.com/Hamster45105/lawnicons-nightly).
> 
> Hope you find it helpful :)

That's great...

Thanks...
